src/PathParser.ts

```diff
@@ -44,7 +44,7 @@ export default class PathParser extends SVGPathData {
 		super(
 			path
 				// Fix spaces after signs.
-				.replace(/[+-.]\s+/g, '-')
+				.replace(/([+-.])\s+/gm, '$1')
```

Comment:
```suggestion
				.replace(/([+\-.])\s+/gm, '$1')
```

?

Reply:
I suppose that symbols **+-.** and **,** can be surrounded by lot of whitespaces so ([+-.])\s+ will trim spaces after all of 4 symbols.

Reply:
there no need to trim spaces around commas

Reply:
Ok I've committed your suggestion.